### ISSUE TITLE
[docs] add equations in `ignite.metrics`

### ISSUE THREAD

**Issue by ydcjeff** [pytorch/ignite#1577]
## 🚀 Feature
Add equations in `ignite.metrics` like PyTorch does on loss functions and also we can revisit the equation later if needed :]

**Comment by vfdev-5** [comment]
@ydcjeff yes, for certain metrics this could be interesting.
